### ISSUE TITLE
All of a sudden: Window has unexpectedly exited... 

### ISSUE THREAD

**Issue by ohddan** [kolton/d2bot-with-kolbot#944]
Worked fine for months and today it starts to exit. The bot opens game, clicks bnet and while connecting: Window has unexpectedly exited... 

And does this over and over again. I can still use it as a loader tho.
Any ideas?

**Comment by noah-** [comment]
This is due to a bug in the CheckRevision code that bnet released yesterday. Certain command line parameters such as -ns can cause this crash. For now, remove -ns.

**Comment by ohddan** [comment]
Did not work :O

**Comment by accept91** [comment]
yeah I have the some for the last couple of days. IT doesn't  even open battle.net
But I have no problems when I try to log in legit

**Comment by ohddan** [comment]
Re-installed everything and bam! Worked.
